config/sync/core.entity_view_display.node.event.full.yml

```diff
@@ -33,11 +33,16 @@ content:
     weight: 3
     region: content
   field_event_date:
-    type: daterange_plain
+    type: date_range_without_time
     label: hidden
     settings:
       timezone_override: ''
+      date_format: 'Y-m-d\TH:i:s'
       separator: '-'
+      one_day: 'd. M Y'
+      one_month: 'd. - {d}. M Y'
+      several_months: 'd. M - {d}. {M} Y'
+      several_years: 'd. M Y - {d}. {M} {Y}'
```

Comment:
:+1: for taking the effort of refactoring the way we show event dates on the full event after introducing `date_range_formatter`.

This removes the need of a large chuck of code currently nested in a template. This is much better.